### DISCUSSION TITLE
Cannot run Turborepo in docker in Github Actions: `IO error: failed to create directory '/app/.turbo/cache'`

### DISCUSSION THREAD

**Discussion by douglasg14b** [vercel/turborepo#8543] [Help]
### Summary

When our docker application attempts to run in Github actions we get the following error:

>  app-1         | > ivr3@1.3.0 start:blackbox /app
> app-1         | > turbo run start:blackbox "--" "--filter" "ivr3-api"
> app-1         | 
> app-1         | 
> app-1         | Attention:
> app-1         | Turborepo now collects completely anonymous telemetry regarding usage.
> app-1         | This information is used to shape the Turborepo roadmap and prioritize features.
> app-1         | You can learn more, including how to opt-out if you'd not like to participate in this anonymous program, by visiting the following URL:
> app-1         | https://turbo.build/repo/docs/telemetry
> app-1         | 
> app-1         |   x IO error: failed to create directory `/app/.turbo/cache`
> app-1         |   |-> failed to create directory `/app/.turbo/cache`
> app-1         |   `-> Permission denied (os error 13)
> app-1         | 
> app-1         |  ELIFECYCLE  Command failed with exit code 1.

This is weird because the user the container is running as not only is the owner of `/app` but an also create directories in `/app` without problems.

If any of the devs on the team run this locally with `docker compose` there are no problems, the problems only present themselves when ran in a GitHub Action.

What may be the cause here? Is turbo running as a different user? Can we turn this behavior off entirely, since we don't need any turbo features when actually running the container?

**Comment by anthonyshew** [comment]
Does running the `turbo` command with the `--env-mode=loose` flag work around this?
